### ISSUE TITLE
Lift desync when looking up (falling through, mismatch position, occasional crash)

### ISSUE THREAD

**Issue by CapoFantasma97** [lucaargolo/lifts#32]
I found that if I look upwards while riding a lift, it causes the lift and player position to desync, which often results in falling through the lift and down, especially so if the ride is decently long (e.g. 50+ blocks). Occasionally the client riding the lift while doing so crashes.

Looking down or straight forward does not cause this issue.

**Comment by vvenaya** [comment]
I fall through the floor after traveling a short distance, looking wherever.  The lift is a deathtrap

**Comment by CapoFantasma97** [comment]
> I fall through the floor after traveling a short distance, looking wherever. The lift is a deathtrap

Yeah sometimes it did the same for me too. I eventually decided it wasn't worth it and just moved on to use the good old vanilla bubble column elevators.

**Comment by Evrey** [comment]
I assume the lift is just too fast for collision checks. A config to make it slower, especially for multiplayer servers, would be helpful.

**Comment by ardissaps** [comment]
Same, I died coz of this bug the platform kinda blinked when reach destination, for now use Create rope pulley Instead, little bit complicated but work.
